client-sdk/go/modules/contracts/types.go

```diff
@@ -133,6 +133,16 @@ type Upgrade struct {
 	Tokens []types.BaseUnits `json:"tokens"`
 }
 
+// ChangeUpgradePolicy is the body of the contracts.ChangeUpgradePolicy call.
+type ChangeUpgradePolicy struct {
+	// ID is the unique instance identifier.
+	ID InstanceID `json:"id"`
+	// UpgradesPolicy is the updated upgrade policy.
+	UpgradesPolicy Policy `json:"upgrades_policy"`
+	// Tokens that should be sent to the contract as part of the change upgrade policy call.
+	Tokens []types.BaseUnits `json:"tokens"`
```

Comment:
Hm, why would one send tokens when updating the policy? The other calls have that when they are running the contract code to allow atomic payments as part of contract code execution.